src/connection.ts

```diff
@@ -24,9 +24,11 @@ export class PostgresJSConnection implements DatabaseConnection {
 
     this.#transaction = createPostgres({...this.#config, options: {...this.#config.options, max: 1}})
 
-    const statement = `start transaction${isolationLevel ? ` ${isolationLevel}` : ''}`
-
-    await this.#transaction.unsafe(statement)
+    if (isolationLevel) {
+      await this.#transaction.unsafe(`start transaction isolation level ${isolationLevel}`)
+    } else {
+      await this.#transaction.unsafe(`begin`)
+    }
```

Comment:
I believe this should allow us to test this like in main repo.

```suggestion
    const compiledQuery = CompiledQuery.raw(
      isolationLevel ? `start transaction isolation level ${isolationLevel}` : 'begin'
    )
    
    await this.executeQuery(compiledQuery)
```

Reply:
Nice! Will update.